code/modules/spells/spell_types/godhand.dm

```diff
@@ -138,3 +138,28 @@
 	M.emote("scream")
 	to_chat(M, span_warning("You feel an explosion of pain erupt in your mind!"))
 	return ..()
+
+/obj/item/melee/touch_attack/raisehand
+	name = "\improper raise bloodman"
+	desc = "The skin of your palm feels thin."
+	on_use_sound = 'sound/magic/wandodeath.ogg'
+	icon_state = "flagellation"
+	item_state = "hivehand"
+	color = "#FF0000"
+/obj/item/melee/touch_attack/raisehand/afterattack(atom/target, mob/living/carbon/user, proximity)
+	var/mob/living/carbon/human/M = target
+	var/mob/living/simple_animal/hostile/asteroid/hivelord/legion/bloodman/L
+	if(!ishuman(M) || M == user || GLOB.bloodmen_list.len > 2)
+		to_chat(M, span_notice("You can't control that many minions!"))
+		return
+	if(NOBLOOD in M.dna.species.species_traits)
+		M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 5)
+		to_chat(M, span_notice("Your head pounds as you raise a bloodman!"))
+	else
+		playsound(M.loc,'sound/items/drink.ogg', rand(10,50), 1)
+		L = new(M.loc)
+		L.stored_mob = M
+		M.forceMove(L)
+		qdel(src)
+		user.blood_volume -= 50 // 9% blood cost, cheaper than the other spell because its not like you can stop near a corpse or find one near you in a fight 
```

Comment:
based accidentally killing yourself by spawning bloodmen with dangerously low blood?

Reply:
yeah, my reasoning for these costs is so that there's consequences to using them that encourage playing safe and getting out of fights. If someone uses these stationside then I want there to be a cost to spamming minions.